src/main/java/com/powsybl/openloadflow/network/extensions/iidm/Tfo3Phases.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2023, Jean-Baptiste Heyberger <jbheyberger at gmail.com>
+ * Copyright (c) 2023, Geoffroy Jamgotchian <geoffroy.jamgotchian at gmail.com>
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow.network.extensions.iidm;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.TwoWindingsTransformer;
+import com.powsybl.openloadflow.util.ComplexMatrix;
+
+/**
+ * @author Jean-Baptiste Heyberger <jbheyberger at gmail.com>
+ */
+public class Tfo3Phases extends AbstractExtension<TwoWindingsTransformer> {
+
+    public static final String NAME = "threePhaseTfo";
+
+    private Boolean isOpenPhaseA1;
+    private Boolean isOpenPhaseB1;
+    private Boolean isOpenPhaseC1;
+    private Boolean isOpenPhaseA2;
+    private Boolean isOpenPhaseB2;
+    private Boolean isOpenPhaseC2;
+
+    // in case of a Delta-Wye or Wye-Delta connexion,
+    // this attribute informs if this is a forward (step-up) or backward (step-down) connexion
```

Comment:
Do you have some documentation about this attribute? I know what is a step-up transformer and a step-down transformer, but I am not sure it is what we mean here.